libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_data_transfer.c

```diff
@@ -202,6 +202,8 @@ struct IotBleDataTransferChannel
 
     bool isUsed;                                  /**< Flag to indicate if the channel is used. */
     bool isOpen;                                  /**< Flag to indicate if the channel is ready to send/receive data. */
+
+    uint8_t ucPeerError : 7;                      /**< 7-bit error status reported by peer */
```

Comment:
I think declaring a bit field does not add additional value here. should we declare as uint8_t instead?

Reply:
The peer error is limited to 7-bits as it's using the previously unused upper 7-bits of CONTROL gatt char -- LSB being the service `ready` bit. I think it helps with readability